sdk/storage/azure-storage-file-share/README.md

```diff
@@ -167,21 +167,21 @@ The URI to reference a share, directory or file must be unique. Within a given s
 If you attempt to create a share, directory, or file with a name that violates naming rules, the request will fail with status code 400 (Bad Request).
 
 ### Share Names
-The rules for File service share names are more restrictive than what is prescribed by the SMB protocol for SMB share names, so that the Blob and File services can share similar naming conventions for containers and shares. The naming restrictions for shares are as follows:
+The rules for File Share service share names are more restrictive than what is prescribed by the SMB protocol for SMB share names, so that the Blob and File services can share similar naming conventions for containers and shares. The naming restrictions for shares are as follows:
```

Comment:
Should be “The rules for File Share service names”

Reply:
Done.